### PR TITLE
Fix text alignment in welcome slides (fixes #1625)

### LINKED ISSUE #1625: Welcome wizard: Text should be left- or right-aligned, not centered
- Started app for the 1st time after install from Google Play Store
- Got: Centered English text
- Expected: left-aligned English text

See e.g. https://practicaltypography.com/centered-text.html

Strings start here: https://github.com/syncthing/syncthing-android/blob/706d4652744737a8593fdd0a2be78d547f0c58d1/app/src/main/res/values/strings.xml#L14

Remember right-to-left-languages

### Version Information

    App Version: v.1.14.0
    Syncthing Version: v1.14.0
    Android Version: 10

### PR DESCRIPTION
**Pull request by offgridauthor**
resolved #1625 changed text alignment from centered to left on three welcome slides

### CODE REVIEW

File: app/src/main/res/layout/activity_firststart_slide3.xml [hunk 1]
```diff
@@ -67,7 +67,7 @@
                 android:paddingRight="@dimen/desc_padding"
                 android:paddingBottom="@dimen/desc_paddingBottom"
                 android:text="@string/location_permission_desc"
-                android:textAlignment="center"
+
```
Comment (imsodin) [inline app/src/main/res/layout/activity_firststart_slide3.xml:70]: Spurious newline.

**Review by imsodin — COMMENTED** [review]
Even if it's straight forward, a screenshot would be appreciated.

And commit message/PR title by convention is "comment (fixes #no)", e.g. "Right align welcome screen text (fixes #1625)" here.

**Review by Catfriend1 — APPROVED** [review]
Looks good from my point of view, but I'm also missing a screenshot from @offgridauthor . Anyway, thank you very much for your contribution.

**Comment by st-review** [comment]
:robot: *beep boop*

I'm going to close this pull request as it has been idle for more than 90 days.

This is not a rejection, merely a removal from the list of active pull requests that is periodically reviewed by humans. The pull request can be reopened when there is new activity, and merged once any remaining issues are resolved.

**Comment by AudriusButkevicius** [comment]
Sorry. We clearly overslept on this. The merge conflicts should be resolved and we should just merge it.

**Comment by imsodin** [comment]
Don't think so: On phone cant check, but I think the other PR fixed it too and there was something unclear to me when i reviewed, that's why I wanted (and if still relevant want) screenshots.

**Comment by AudriusButkevicius** [comment]
Actually, you are right, this seems fixed, hence closing.

**Pull request closed without merging**